.travis.yml

```diff
@@ -22,23 +24,23 @@ branches:
     - master
 
 before_script:
+  - unset JULIA_PROJECT
```

Comment:
Do we perhaps also need this `unset` inside of `script:` and/or `after_script`? (Since tests on Travis seem to have passed: apparently not?!)

Reply:
This seems to be valid for all parts of the tests. I only had one additional export before, too, also in `before_script`, so this seems to be valid for all parts of the test.